mms_msg/sampling/environment/sro.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from mms_msg.sampling.utils.rng import get_rng
+import paderbox as pb
+
+
+def sample_sro_weights(example, num_nodes, sto_range, sro_range):
+    """
+    Samples the average sapling rate offset and sampling time offset for an example
+    """
+    rng = get_rng(example['example_id'])
+    avg_sro = rng.uniform(*sro_range, size=num_nodes)
+    sto = rng.randint(*sto_range, size=num_nodes)
```

Comment:
If you want these to be deterministic even when `num_nodes` changes, use `get_rng_example(example, 'sto').randint(...)`